src/navigation/_AppSettingsContext.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+/**
+ * Creates an app settings context to share global app setting with the dev toolbar.
+ */
+const AppSettingsContext = React.createContext({});
+
+/**
+ * Helper to add the app settings context as a prop to teh wrapped component.
```

Comment:
teh -> the

Reply:
You let me down spellchecker
![](https://media.giphy.com/media/1JyWrrkCIUQyQ/giphy.gif)